.circleci/config.yml

```diff
@@ -48,20 +48,20 @@ jobs:
       - run:
           name: init .pypirc
           command: |
-            echo -e "[pypi]" >> ~/.pypirc
-            echo -e "username = $PYPI_USERNAME" >> ~/.pypirc
-            echo -e "password = $PYPI_PASSWORD" >> ~/.pypirc
+            python .circleci/create_pypirc.py \
```

Comment:
Cool. I guess this is necessary for the test pypi deploy?

Reply:
I was getting auth errors for TestPyPi I looked up examples of .pypirc files that featured TestPyPi deployment and they were more involved. I could still write all the lines here but after a point I figured it'd be easier in a template file.